docs/modeling/custom-type.md

```
@@ -40,6 +41,33 @@ type UserProfile {
 }
 ```
 
+### Relation Fields
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)